optuna/README.md

```diff
@@ -1,16 +1,124 @@
-# Optuna minimum example
+# Optuna-Based Design Space Exploration (DSE)
 
-Use `--define X=Y` to override variables in ORFS `config.mk`. `--define X=Y` is adequate for simple ORFS flows to drive Optuna where there's no ambiguity as to which of several subflows that varibles such as `PLACE_DENSITY` refers to:
+Intelligent parameter optimization using Optuna to find optimal design parameter combinations that meet timing constraints.
 
-    bazelisk run --define PLACE_DENSITY=0.1234 //:lb_32x128_floorplan_deps /tmp/x print-PLACE_DENSITY
+## Experiment Design
 
-Outputs:
+**Goal**: Find optimal CORE_UTILIZATION and PLACE_DENSITY at a fixed clock frequency to minimize area (or area + power).
 
-    PLACE_DENSITY = 0.1234
+**Why Optuna?**
```

Comment:
- There are many Python hyperparameter search frameworks, Optuna shines in simplicity and good results for limited effort. No need install or configure a server for remote execution.